release-notes-2.1.0.md

```diff
@@ -5,4 +5,9 @@ ${version-number}
 
 #### New Features
 
+#### Bug Fixes
+- **I887031:** Providing a Trust Manager when connecting to RabbitMQ server via TLS. Trust Manager factory
```

Comment:
Reword to

Replace use of the TrustEverythingTrustManager with a TrustManager provided by the javax.net.ssl.TrustManagerFactory.